### ISSUE TITLE
Add support for MixMHCpred?

### ISSUE THREAD

**Issue by malachig** [griffithlab/pVACtools#240]
Investigate adding support for this predictor that uses mass spec data to train unlike the other algorithms we currently support.
https://github.com/GfellerLab/MixMHCpred

**Comment by EdGreen21** [comment]
The Gfeller lab has now added an HLAII predictor:

https://github.com/GfellerLab/MixMHC2pred

**Comment by malachig** [comment]
Thanks for the update @EdGreen21 !

**Comment by MartinChiaHsinLiu** [comment]
Hi,
I just install pVactools after 2019.Sept.  
However, it still does not support MixMHCpred or MixMHC2pred
I had run the pip install pvactools --upgrade

my command:
pvacseq run my.vep.tx.vcf SRR2602447 HLA-A*02:01 MixMHCpred,MixMHC2pred /home/liu/RNA_seq/output/DNA/ -e 8,9,10,11 --pass-only --normal-sample-name SRR2602449

Error return:
pvacseq run: error: argument prediction_algorithms: invalid choice: 'MixMHCpred,MixMHC2pred' (choose from 'MHCflurry', 'MHCnuggetsI', 'MHCnuggetsII', 'NNalign', 'NetMHC', 'NetMHCIIpan', 'NetMHCcons', 'NetMHCpan', 'PickPocket', 'SMM', 'SMMPMBEC', 'SMMalign')

**Comment by susannasiebert** [comment]
You are correct that support for MicMHCpred has not been added yet. While we would love to add it to pVACtools, unfortunately, it is not python-compatible at the moment. Please subscribe to this issue if you wish to be kept in the loop for any future development.

**Comment by MartinChiaHsinLiu** [comment]
Thanks for a response quickly.

Best Regards,
Martin

**Comment by susannasiebert** [comment]
@MartinChiaHsinLiu Support for MixMHCpred has just been added in our latest version 7.0.0. Please give it a try and let us know if you run into any issues.
